README.md

```diff
@@ -2,13 +2,96 @@
 
 ![](Extras/img.jpg)
 
-## Python / Raspberry Pi interpretation of Bryan Boyer's Very Slow Movie Player  
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
+> Very Slow Movie Player using Python + Raspberry Pi
```

Comment:
I was reading the guide for the Standard README format (nice find by the way, haven't used that before) and saw this: https://github.com/RichardLitt/standard-readme/blob/master/spec.md#short-description 

Should the ```>``` be removed here if this is technically the "short description"? See bullet 3 in that link.

Reply:
It looks like [in the past](https://github.com/RichardLitt/standard-readme/commit/2986c499017e94aa991e93180e717f5a80e44ca2#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R40) it's required that it starts with `>` and [they changed it](https://github.com/RichardLitt/standard-readme/commit/0211114475754c064f643b6e36b382e9c9298312#diff-bc6661da34ecae62fbe724bb93fd69b91a7f81143f2683a81163231de7e3b545R81). Fixed in 06b1963, good catch, Rob